tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/util/JMSClusteredTestBase.java

```diff
@@ -119,7 +118,7 @@ private void setupServer2() throws Exception {
 
       JMSConfigurationImpl jmsconfig = new JMSConfigurationImpl();
 
-      mBeanServer2 = MBeanServerFactory.createMBeanServer();
+      mBeanServer2 = getMBeanServer();
```

Comment:
As in other cases, misleading having 2 variables both using the same actual MBeanServer